### PR TITLE
Added the tier name, current and max level message

### PR DESCRIPTION
**Pull request by king-glitch**
Added the tier name, current and max level message

### CODE REVIEW

**Comment by Guillaume-Lebegue** [comment]
Thanks for doing that, kinda forgot about it when developing it.
I'm ok with most of the changes.
However, I'm not ok with the update from " to ' in the local file as the rest of BentoBox is using "

**Comment by king-glitch** [comment]
> Thanks for doing that, kinda forgot about it when developing it.
> I'm ok with most of the changes.
> However, I'm not ok with the update from " to ' in the local file as the rest of BentoBox is using "

Sorry, it was my auto format. And I'm very newbie on Java. Thank you for teaching me good pratices!

File: src/main/resources/locales/en-US.yml [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 
 protection:
-  flags:
-    UPGRADES_RANK_RIGHT:
-      name: "Upgrade Rank"
-      description: "Switch who can upgrade"
+    flags:
+        UPGRADES_RANK_RIGHT:
+            name: "Upgrade Rank"
+            description: "Switch who can upgrade"
```
Comment (Guillaume-Lebegue) [inline src/main/resources/locales/en-US.yml]: That's close,
YAML advises to use 2 spaces for indentation

If you want, you can use websites like http://www.yamllint.com/ to check .yml files

**Review by Guillaume-Lebegue — CHANGES_REQUESTED** [review]
Don't worry about that.
Just a little thing before the approval

**Review by Guillaume-Lebegue — APPROVED** [review]
Thx, that's good for me.

**Pull request merged**